pyproject.toml

```diff
@@ -215,9 +215,9 @@ ignore = [
     "docs/**",
     "examples/**",
     "tests/**",
-    "tmt/checks/*.py",
+    # "tmt/checks/*.py",
```

Comment:
Why not just remove them from the `ignore` list?

Reply:
Fixed during rebase.